.github/workflows/task-test.yml

```diff
@@ -56,10 +55,16 @@ jobs:
       run:
         shell: bash -l -eo pipefail {0}
     steps:
-      - name: Pre-steps Dependencies
-        if: inputs.pre-steps-script
+      # Split to alpine and non-alpine due to different default shells, once the dependency installation is done, we can use the same shell in the rest of the flow
```

Comment:
Do you know what commands are actually shell-specific in the script?

Reply:
alpine does not come with bash installed, it comes with sh. so when calling bash, it will fail.